corehq/apps/app_manager/models.py

```diff
@@ -4837,6 +4909,7 @@ def create_profile(self, is_odk=False, with_media=False, template='app_manager/p
         if toggles.CUSTOM_PROPERTIES.enabled(self.domain) and "custom_properties" in self__profile:
             app_profile['custom_properties'].update(self__profile['custom_properties'])
 
+        locale = self.langs[0] if not build_profile_id else self.build_profiles[build_profile_id].langs[0]
```

Comment:
It looks like `self.langs` might reasonably be empty too, at least during testing.  This is handled [here](https://github.com/dimagi/commcare-hq/pull/11295/files#diff-535390eb5c83a4cd3449f82afd1fa9fbR4764):
```
return self.langs[0] if len(self.langs) > 0 else "en"
```
@calellowitz do you know if this is possible through the UI?